ThatDefaultProject/Assets/ThatDefaultProject/Scripts/LookAtCamera.cs

```diff
@@ -0,0 +1,56 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace that
+{
+    public class LookAtCamera : MonoBehaviour
+    {
+        [Tooltip("If no object is assigned, it will look for an object that has a camera.")]
+        [SerializeField] private GameObject _cameraObject;
+        private Camera _camera;
+        [SerializeField] private bool _lockZRotation = false;
+        [SerializeField] private bool _lockXRotation = true;
+        [SerializeField] private bool _flipAroundY = false;
+
+        private Vector3 _lookPos = new();
+        private bool _isInitialized = false;
+
+        void Start()
+        {
+            if (_cameraObject == null)
+            {
+                _camera = FindAnyObjectByType<Camera>();
```

Comment:
Maybe we should check for Camera.main as well?

Reply:
Good point!